### PR TITLE
Add failing test for cyrillic sentence length (issue #139)

### PR DESCRIPTION
**Pull request by AjBreidenbach**

### CODE REVIEW

File: test.py [hunk 1]
```diff
@@ -89,6 +89,48 @@
...
+    "платить по счетам и не отставать от «Джонсов», что "
+    "пренебрегают радостью жизни; удовольствием, которое может "
+    "быть наградой за расслабляющую игру с другим человеком. "
+    "Было сказано, что «человек - это для того, чтобы иметь "
+    "радость», но слишком часто мы идем по жизни без особой "
+    "радости. Игры позволяют нам расслабиться, узнать что-то "
+    "новое и интересное, взаимодействовать с людьми на другом, "
+    "более комфортном уровне и наслаждаться безопасным "
+    "соревнованием. По этим причинам взрослые должны уделять "
+    "больше внимания играм в своей жизни"
```
Comment (alxwrd) [inline test.py:131]: The final sentence doesn't end with a full stop, is that right?

**Review by alxwrd — COMMENTED** [review]
@AjBreidenbach, thank you for this. I've highlighted a (maybe) missing full stop at the end of the last sentence.

File: test.py (hunk 1 above)
Comment (AjBreidenbach) [reply test.py:134]: That's right.  It's a translation of `long_test` with the only formatting change being the guillemets ( '«' and '»' ).
I messed up with the first commit, but it's fixed, and the test passed with my `set_lang` implementation.

Comment (alxwrd) [reply test.py:134]: :+1:

**Comment by alxwrd** [comment]
This is great, thank you. I'll hold off merging for now until I've investigated potential fixes. Either your fork, or a new regex.

**Comment by alxwrd** [comment]
Merging this as we already have some failing tests

**Pull request merged**